auth/tests/unit/cognito-federation-throttles-cloudwatch-alarm.test.ts

```diff
@@ -0,0 +1,227 @@
+import { describe, it, beforeAll, expect } from "vitest";
```

Comment:
Suggestion - could we combine or parameterize the tests so that we have 1 file for all the alarms?

Reply:
I was thinking this while writing them and did start doing some digging and got lost in JFDI - I'll look further into this in my next PR which will include the int and acc tests.